includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -105,7 +105,7 @@ public function read( &$coupon ) {
 			'excluded_product_ids'        => array_filter( (array) explode( ',', get_post_meta( $coupon_id, 'exclude_product_ids', true ) ) ),
 			'usage_limit'                 => get_post_meta( $coupon_id, 'usage_limit', true ),
 			'usage_limit_per_user'        => get_post_meta( $coupon_id, 'usage_limit_per_user', true ),
-			'limit_usage_to_x_items'      => get_post_meta( $coupon_id, 'limit_usage_to_x_items', true ),
+			'limit_usage_to_x_items'      => 0 < get_post_meta( $coupon_id, 'limit_usage_to_x_items', true ) ? get_post_meta( $coupon_id, 'limit_usage_to_x_items', true ) : null,
```

Comment:
That's what I figured, but I didn't know whether we wanted to allow users to make coupons like that. :)

Good to go!